views/partials/header.ejs

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
I would add a nav bar to the header with links to make it easier to move around the site